### ISSUE TITLE
Pretrained model selection

### ISSUE THREAD

**Issue by sehunfromdaegu** [Jwoo5/fairseq-signals#23]
Hi,

I couldn't find information on how many epochs are needed for the pretraining model in your paper. Is it same as in the config in this repository?

Additionally, are downstream tasks performed using the last checkpoint of the pretrained model, or is the best epoch checkpoint used?

Thank you.

**Comment by Jwoo5** [comment]
1. yes, the number of epochs that we used for pretraining is the same with the specified epochs in each config file.
2. we used the best epoch checkpoint where the relevant metric for the validation set is the best. you can see that the configurations for this option are commented out in the config files.

**Comment by sehunfromdaegu** [comment]
Thanks
